packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -125,9 +126,12 @@ export default class UIForm extends React.Component {
 					}
 				});
 				propertyName = schema.key[schema.key.length - 1];
+				const formId = this.props.id;
+				this.onTrigger(event, { formData, formId, propertyName, value });
+			} else {
+				const payload = { trigger: schema.triggers[0], schema, properties: formData };
+				this.onTrigger(event, payload);
```

Comment:
Same here. No need to define a constant if it's to use it only once right after.